openstack/telemetry/v2/meters/results.go

```diff
@@ -0,0 +1,99 @@
+package meters
+
+import (
+	"reflect"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+)
+
+type MeterListResult struct {
```

Comment:
In light of `listResult` below, how about just `MeterResult`? Unless there are different `struct` formats for different meter operations?